media/livestream/api/LiveStream.Samples.Tests/LiveStreamFixture.cs

```diff
@@ -50,26 +65,99 @@ public LiveStreamFixture()
 
     public async Task InitializeAsync()
     {
+        await CleanOutdatedResources();
+
         TestInputId = $"{InputIdPrefix}-{RandomId()}";
         InputIds.Add(TestInputId);
         TestInput = await _createInputSample.CreateInputAsync(ProjectId, LocationId, TestInputId);
+
+        TestUpdateInputId = $"{InputIdPrefix}-updated-{RandomId()}";
+        InputIds.Add(TestUpdateInputId);
+        TestUpdateInput = await _createInputSample.CreateInputAsync(ProjectId, LocationId, TestUpdateInputId);
+
+        TestChannelId = $"{ChannelIdPrefix}-{RandomId()}";
+        ChannelIds.Add(TestChannelId);
+        TestChannel = await _createChannelSample.CreateChannelAsync(ProjectId, LocationId, TestChannelId, TestInputId, ChannelOutputUri);
     }
 
-    public async Task DisposeAsync()
+    public async Task CleanOutdatedResources()
```

Comment:
I'd like to know why were this channels/inputs left behind in the first place. You are already deleting all channels/inputs you created on Dispose so this should never have anything to delete, right?

Reply:
Correct. Possibly the entire test timed out, leaving the channels and inputs.